services/ir/artifacts.mk

```diff
@@ -31,14 +31,22 @@ get.cli:
 	@mkdir -p ./vendor
 
 ifeq (${NEOFS_CLI_PATH},)
-	@echo "⇒ Download NeoFS CLI binary from ${NEOFS_CLI_URL}"
-	@curl \
-		-sSL "${NEOFS_CLI_URL}" \
-		-o ${NEOFS_CLI_ARCHIVE_FILE} 
-	@tar -xvf ${NEOFS_CLI_ARCHIVE_FILE} -C ./vendor | xargs -I {} \
-		mv ./vendor/{} ${NEOFS_CLI_FILE}
-	@rm ${NEOFS_CLI_ARCHIVE_FILE}
+    ifneq (${NEOFS_CLI_URL},)
+		@echo "⇒ Download NeoFS CLI binary from ${NEOFS_CLI_URL}"
+		@curl \
+		    -sSL "${NEOFS_CLI_URL}" \
+		    -o ${NEOFS_CLI_ARCHIVE_FILE}
+		@tar -xvf ${NEOFS_CLI_ARCHIVE_FILE} -C ./vendor | xargs -I {} \
+			mv ./vendor/{} ${NEOFS_CLI_FILE}
+		@rm ${NEOFS_CLI_ARCHIVE_FILE}
+    else
+	    @echo "⇒ NEOFS_CLI_URL is empty."
+    endif
 else
-	@echo "⇒ Copy local binary from ${NEOFS_CLI_PATH}"
-	@cp ${NEOFS_CLI_PATH} ${NEOFS_CLI_FILE}
+    ifneq (${NEOFS_CLI_PATH},)
+		@echo "⇒ Copy local binary from ${NEOFS_CLI_PATH}"
+		@cp ${NEOFS_CLI_PATH} ${NEOFS_CLI_FILE}
+    else
+		@echo "⇒ NEOFS_CLI_PATH is empty."
```

Comment:
While this can never happen.

Reply:
> While this can never happen.

This will happen if the user does not specify NEOFS_CLI_URL and NEOFS_CLI_PATH in the .env file.
I think that these messages help the user to understand faster what went wrong when building the dev-env.

Reply:
It won't. He will end up in the `NEOFS_CLI_PATH and NEOFS_CLI_URL are empty` case.